.github/workflows/reusable.yml

```diff
@@ -0,0 +1,104 @@
+name: Deploy AKS-Construction
+
```

Comment:
Do we need to think about adding some docs to show people how to consume the workflow?

Reply:
Lets put a pin in this one. Captured in #317